public/index.html

```diff
@@ -24,6 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://component-vishalg8454.netlify.app/components/main.css">
+    <script src="https://kit.fontawesome.com/06bd83eb3a.js" crossorigin="anonymous"></script>
     <title>React App</title>
```

Comment:
Change the title of the website with your website name

Reply:
Sure, will do.